data/strings/types_strings.txt

```diff
@@ -21798,6 +22198,46 @@
     zh-Hans = 室外设备
     zh-Hant = 室外設備
 
+  [type.shop.outpost]
+    en = Pickup Point
```

Comment:
Online orders pickup

Reply:
'Retail Collection Point' may be better, also worth noting that they are called 'Click and Collect points' in the UK

Reply:
'it = Negozio HiFi' would be better instead of 'it = Audio ad alta fedeltà'. No one uses that.